apps/freeablo/main.cpp

```diff
@@ -350,14 +417,31 @@ void runGameLoop(const bpo::variables_map& variables)
     
     std::pair<size_t, size_t> destination = player->mPos.current();
     
-    bpt::ptree hotkeypt;
     Misc::readIni("resources/hotkeys.ini", hotkeypt);
     
-    quit_key = hotkeypt.get<int>("Hotkeys.quit");
-    noclip_key = hotkeypt.get<int>("Hotkeys.noclip");
-    changelvldwn_key = hotkeypt.get<int>("Hotkeys.changelvldwn");
-    changelvlup_key = hotkeypt.get<int>("Hotkeys.changelvlup");
-
+    quit_key[0] = hotkeypt.get<int>("Quit.key");
```

Comment:
Im not sure this is the place to put this code, it might be better to create some sort of hotkey management class to contain it.
Also, again, it would be better to factor out the loading of one hotkey into a function.